### DISCUSSION TITLE
SQLalchemy 1.4 new async functions

### DISCUSSION THREAD

**Discussion by jcf-dev** [fastapi/fastapi#6734] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorials in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

SQLalchemy > 1.4.x seems not to work with the [FastAPI Async SQL Tutorial](https://fastapi.tiangolo.com/advanced/async-sql-databases/) anymore. The tutorial code is working if I use versions < 1.3.x. 

I would greatly appreciate it if you could show me how to integrate the new Async capable SQLalchemy 1.4.x to FastAPI. I have read the docs [here](https://docs.sqlalchemy.org/en/14/orm/extensions/asyncio.html), tried it but can't get it working with FastAPI. I'm lost, and I don't know what I'm doing. 😅

**Comment by raphaelauv** [comment]
Changelog of 1.4 -> https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#asynchronous-io-support-for-core-and-orm

with breaking changes from 1.3

**Comment by ycd** [comment, marked as answer]
See: https://github.com/tiangolo/fastapi/pull/2331

**Comment by jcf-dev** [comment]
> See: https://github.com/tiangolo/fastapi/pull/2331

Nice! Thanks! Exactly what I needed.

**Comment by tiangolo** [comment]
Thanks for the help here @raphaelauv and @ycd ! 👏  🙇 

Thanks for reporting back and closing the issue @jcf-dev  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
